packages/node_modules/@webex/plugin-meetings/src/common/errors/webex-errors.js

```diff
@@ -0,0 +1,95 @@
+// The document would auto generate the doc for errors
+
+// 1) Error codes for Oauth, login, mercury should be separated out (Code range)
+// 2) Errors from the actual locus server or other server in case, we can use the same error code as locus and redirect it
+// 3) Any errors we generate from the SDK can be split into various categories
+// Parameter validation, user action, connection, media specific(They can have different range)
+
+
+// TODO: all the errors from the server need to be captured
+
+// add a way to log and send metrics if needed
+const WebexMeetingsError = require('./webex-meetings-error');
+
+const WebExMeetingsErrors = {};
+
+/**
+ * Create a {@link WebexMeetingsError} for a given code and message.
+ * @private
+ * @param {number} [code] - Error code
+ * @param {string} [message] - Error message
+ * @returns {WebexMeetingsError}
+ */
+exports.createMeetingsError = function createMeetingsError(code, message) {
```

Comment:
Was there a reason behind using the `exports.` method instead of just `export function`?

Reply:
no specific reason, i dont know if we can do a mix of export function and export.

THe exports are classes so wont fit that model

Reply:
```suggestion
export function createMeetingsError(code, message) {
```